tmc-langs-util/src/main/java/fi/helsinki/cs/tmc/langs/util/ProjectType.java

```diff
@@ -0,0 +1,21 @@
+package fi.helsinki.cs.tmc.langs.util;
+
+import fi.helsinki.cs.tmc.langs.LanguagePlugin;
+import fi.helsinki.cs.tmc.langs.ant.AntPlugin;
+
+/**
+ * All the possible project types
+ */
+public enum ProjectType {
```

Comment:
Ideally I'd use Guice multibindings or something equivalent here to get a true plugin architecture. Enums are inflexible.

It's OK to do this later if you want.